standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2025,7 +2027,9 @@ struct CreateTableRequest {
    7: optional list<SQLDefaultConstraint> defaultConstraints,
    8: optional list<SQLCheckConstraint> checkConstraints,
    9: optional list<string> processorCapabilities,
-   10: optional string processorIdentifier
+   10: optional string processorIdentifier,
+   // use boolean skipFSWrites to decide whether create directory in file system or not
```

Comment:
Add "HIVE-26012" in the comment so that if anyone needs more details on this, they can review the jira.

Reply:
Sure.